ui/home_page_spec.rb

```diff
@@ -1,24 +1,25 @@
 # frozen_string_literal: true
 
+require_relative './spec_helper'
```

Comment:
you get this automatically via .rspec

Reply:
From my understanding this would mean using the application root spec/spec_helper, all the necessary dependencies for the ui tests are in a separate spec_helper. If the ui directory is copied over in the qa test process only, then the other spec directory will not be found and dependencies will not be loaded. Im happy to adjust the current spec_helper for just the ui tests if you see fit. Other than that without the spec_helper require in the ui specs the tests fail to run, as you would expect given the dependencies are not found.